backend/tasks/celery_tasks.py

```diff
@@ -151,6 +154,20 @@ def parse_replay_gcp(self, fn):
     r = requests.post(GCP_URL, data=encoded_file, timeout=0.5)
 
 
+@celery.task(base=DBTask, bind=True, priority=9)
+def parse_replay_gcp(self, fn):
+    with open(fn, 'rb') as f:
+        encoded_file = base64.b64encode(f.read())
+    r = requests.post(GCP_URL, data=encoded_file)
+    response = r.json()
+    i = io.BytesIO(base64.b64decode(response['proto']))
+    protobuf_game = ProtobufManager.read_proto_out_from_file(i)
+    i2 = io.BytesIO(base64.b64decode(response['pandas']))
```

Comment:
pls.

I feel `r` does not need to be created - just chain `.json()` on.

`i` and `i2` can definitely do with better names.

Reply:
Old code, I swear I've changed ;)